### DISCUSSION TITLE
FEATURE｜GH Actions针对不同平台分开构建

### DISCUSSION THREAD

**Discussion by shenjackyuanjie** [MCSLTeam/MCSL2#76] [功能建议] [answered]
https://github.com/shenjackyuanjie/Difficult-Rocket/actions/workflows/nuitka.yml
可以参考我这里的nuitka构建
建议是nuitka的构建命令每个平台分离，这样可以针对平台进行特殊修改
以及建议单独写一份requrements.txt 提升构建速度

（其实我本来想直接发PR的，但是高一，没时间，先留个issue，有时间了发PR（

**Comment by lxhtt** [comment]
周末我试试

**Comment by lgc2333** [comment]
resolved in #17 (maybe)

**Comment by lxhtt** [comment, marked as answer]
已添加并完善。

**Comment by shenjackyuanjie** [comment]
:thinking:
反正现在还是我的啦！（
